.travis.yml

```diff
@@ -8,3 +8,4 @@ before_install:
 script:
   - xcodebuild -workspace ooniprobe.xcworkspace/ -scheme ooniprobe CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO | xcpretty
   - xcodebuild clean test -workspace ooniprobe.xcworkspace/ -sdk iphonesimulator -scheme OONIProbeUITests -destination 'name=iPhone SE' CODE_SIGNING_REQUIRED=NO | xcpretty
+  - xcodebuild -workspace ooniprobe.xcworkspace/ -scheme ooniprobe -sdk iphonesimulaton -destination 'name=iPhone SE'   test  | xcpretty
```

Comment:
Got it from https://www.mokacoding.com/blog/running-tests-from-the-terminal/ but not sure it's what I want

Reply:
@hellais @bassosimone is this working as expected in Travis? 
I have absolutely no idea if I wrote the correct command.

Reply:
@hellais this

Reply:
I ran this command on my machine before pushing it and worked.
I know nothing about travis and syntax this is why I asked your help

Reply:
Could `pipefail` be helpful here?

Reply:
I tried to make some screenshots today and it failed in languages non en.
I'm gonna file a ticket because it has to be fixed soon. In the meantime I guess this pr can be merged as the `OONIProbeUITests` is already in master so doesn't affect this branch

Reply:
https://github.com/ooni/probe-ios/issues/262